pkgs/development/python-modules/galois/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pytestCheckHook
+, numpy
+, numba
+, typing-extensions
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "galois";
+  version = "0.0.32";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "mhostetter";
+    repo = "galois";
+    rev = "bf1275a815fd21162198ce788633294ad97b4675";
+    sha256 = "sha256-+cxRLrfqk3N9pWKCVsTxruZwMYZ5dQyKJRnrb8y+ECM=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    numba
+    typing-extensions
+  ];
+
+  checkInputs = [
+    pytest
+    pytestCheckHook
+  ];
+
+  postPatch = ''
+     substituteInPlace setup.cfg \
+       --replace "numpy >= 1.18.4, < 1.23" "numpy >= 1.18.4, < 1.24"
```

```suggestion
       --replace "numpy >= 1.18.4, < 1.23" "numpy >= 1.18.4"
```

Without an upper bound we can just rely on the tests and don't have to keep on patching here, when we upgrade numpy.